GitUI/Infrastructure/Telemetry/DiagnosticsClient.cs

```diff
@@ -1,5 +1,5 @@
 ﻿// The original idea and the implementation are borrowed from  https://github.com/NuGetPackageExplorer/NuGetPackageExplorer
-// Credits to Oren Novotny
+// Credits to Claire Novotny
```

Comment:
```suggestion
// Credits to https://github.com/clairernovotny
```

Reply:
Can you explain why this change helps?

Reply:
People do change their names, pronouns, etc., hence it is preferred to use GitHub aliases because these represent the online identity of each contributor (this came up at one of training courses). 
If I have originally put https://github.com/onovotny we would unlikely be having this conversation.